server/views/pages/createGoals/index.njk

```diff
@@ -123,28 +123,29 @@
                 errorMessage: errors | findError('targetCompletionDate')
               }) }}
 
-            <div class="govuk-form-group">
-              <!-- TODO: Populate the stepNumber object to relect the correct number on the UI -->
-              {% set stepTitleLabel = 'Step ' + form.stepNumber %}
-              <!-- TODO: Populate the stepTitle object to reflect the correct step title on the UI -->
-              {{ govukCharacterCount({
-                name: "stepTitle",
-                id: "stepTitle",
-                value: form.stepTitle,
-                maxlength: 512,
-                rows: 2,
-                label: {
-                  text: stepTitleLabel,
-                  classes: "govuk-label--s",
-                  isPageHeading: false
-                },
-                hint: {
-                  text: "Describe this step"
-                },
-                errorMessage: errors | findError('stepTitle')
-              }) }}
-
-            </div>
+            {% for step in form.steps %}
+              <div class="govuk-form-group">
+                <!-- TODO: Populate the stepNumber object to relect the correct number on the UI -->
```

Comment:
Can we not resolve these todo's by using the loop index?

Reply:
Currently we have no steps to loop through in the session, so the loop isn't actually working - I've removed it for now and just put placeholders in until we have some session data